transcript/src/test/java/com/transcriptanalyzer/transcript/TreeTest.java

```diff
@@ -1,48 +1,72 @@
         ArrayList<ArrayList<ArrayList<String>>> testData = new ArrayList<>();
 
-        ArrayList<String> turn1 = new ArrayList<String>();
-        ArrayList<String> turn2 = new ArrayList<String>();
-        ArrayList<String> turn3 = new ArrayList<String>();
-        ArrayList<String> turn4 = new ArrayList<String>();
+        // init test data for transcript 1
+        ArrayList<String> turn1T1 = new ArrayList<String>();
+        ArrayList<String> turn2T1 = new ArrayList<String>();
+        ArrayList<String> turn3T1 = new ArrayList<String>();
+        ArrayList<String> turn4T1 = new ArrayList<String>();
 
+        // init test data for transcript 2
+        ArrayList<String> turn1T2 = new ArrayList<String>();
+        ArrayList<String> turn2T2 = new ArrayList<String>();
+        ArrayList<String> turn3T2 = new ArrayList<String>();
+        ArrayList<String> turn4T2 = new ArrayList<String>();
+
+        // init test data for transcript 3
+        ArrayList<String> turn1T3 = new ArrayList<String>();
+        ArrayList<String> turn2T3 = new ArrayList<String>();
+
+        // init test data for transcript 1
         ArrayList<ArrayList<String>> transcript1 = new ArrayList<ArrayList<String>>();
         ArrayList<ArrayList<String>> transcript2 = new ArrayList<ArrayList<String>>();
         ArrayList<ArrayList<String>> transcript3 = new ArrayList<ArrayList<String>>();
 
-        turn1.add("Hi, welcome to Generic Pizza Place (tm)! What can I help you with today?");
-        turn1.add("Remake");
-        transcript1.add(turn1);
-        turn2.add("Sorry, could you rephrase that for me?");
-        turn2.add("Order");
-        transcript1.add(turn2);
-        turn3.add("We have 3 different kinds of pizzas available with one being on special per day. The choices are chesse, veggie, and pepperoni.");
-        turn3.add("Veggie");
-        transcript1.add(turn3);
-        turn4.add("We will make you a veggie pizza. Please come pick it up in 20 minutes.");
-        transcript1.add(turn4);
-        transcript2.add(turn1);
-        transcript2.add(turn2);
-        turn3.set(1, "Pepperoni");
-        transcript2.add(turn3);
-        turn4.set(0, "We will make you a pepperoni pizza. Please come pick it up in 20 minutes.");
-        transcript1.add(turn4);
+        // creating first transcript
+        turn1T1.add("Hi, welcome to Generic Pizza Place (tm)! What can I help you with today?");
+        turn1T1.add("Remake");
+        transcript1.add(turn1T1);
+        turn2T1.add("Sorry, could you rephrase that for me?");
+        turn2T1.add("Order");
+        transcript1.add(turn2T1);
+        turn3T1.add("We have 3 different kinds of pizzas available with one being on special per day. The choices are chesse, veggie, and pepperoni.");
+        turn3T1.add("Veggie");
+        transcript1.add(turn3T1);
+        turn4T1.add("We will make you a veggie pizza. Please come pick it up in 20 minutes.");
+        transcript1.add(turn4T1);
 
```

Comment:
For future testing purposes, it may be worth creating a function which could take in an array of strings and automatically generate these mock transcripts - would clean up the file and make it a lot less tedious to write new tests! Great job on implementing such thorough testing though.